src/modules/controls/Controls.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { node } from "prop-types";
+import { useClassNames } from "src/hooks/useClassNames";
+import { Control } from "./Control";
+
+function Controls({ children, ...props }) {
+  const classNames = useClassNames('controls relative flex -mt-10 group-hover:mt-0 transition-mt duration-200', props);
+
+  return (
+    <div className={classNames}>
+      {children}
+    </div>
+  )
+}
+
+Controls.Control = Control;
+
+Controls.propTypes = {
+  children: node.isRequired,
+};
+
+export { Controls };
```

Comment:
🟢  by default controls are hidden until the group is hovered. I should change this to be based on a prop to animate the controls or even move the animation logic to the `FileCard` component. I'll give it a think.